layouts/organization/single.html

```diff
@@ -1,42 +1,114 @@
+                        {{ if isset . "email"}}<a href="mailto:{{ .email }}"><img width="25" src="/about/email_icon.png"></a>{{ end }}
+                        <br>
+                        {{ end }}
+                    </p>
+                </div>
+            </li>
+            <li class="flex-item">
+                <h1>GNU Radio 4.0</h1>
+                <div style="text-align: left;">
+                <p style="line-height: 1.4;">
+                Lead:<br>
+                Josh Morman<br><br>
+                Members:<br>
+                Bastian Bloessl<br>
+                Marcus Müller<br>
+                </div>
                 </p>
-            </div>
-            
-        </div>
+            </li>
+            <li class="flex-item">
+                <h1>GRC</h1>
+                <div style="text-align: left;">
+                    <p style="line-height: 1.4;">
+                    Lead:<br>
+                    TBD<br><br>
+                    Members:<br>
+                    Håkon Vågsether<br>
+                    </div>
+                    </p>
+            </li>
+            <li class="flex-item">
+                <h1>Documentation</h1>
+                <div style="text-align: left;">
+                    <p style="line-height: 1.4;">
+                    Lead:<br>
+                    Marc Lichtman<br><br>
+                    Members:<br>
+                    Barry Duggan<br>
+                    </div>
+                    </p>
+            </li>
+            <li class="flex-item">
+                <h1>GRCon</h1>
+                <div style="text-align: left;">
+                    <p style="line-height: 1.4;">
+                    Lead:<br>
+                    Samantha Palazzolo<br><br>
+                    Members:<br>
+                    Josh Morman<br>
+                    Marc Lichtman<br>
+                    Barry Duggan<br>
+                    Derek Kozel<br>
+                    </div>
+                    </p>
+            </li>
+            <li class="flex-item">
+                <h1 style="margin-bottom: 0px;">SigMF</h1>
+                <h4>(Signal Metadata Standard)</h4>
+                <div style="text-align: left;">
+                    <p style="line-height: 1.4;">
+                    Lead:<br>
+                    Jacob Gilbert<br><br>
+                    Members:<br>
+                    Marc Lichtman<br>
+                    </div>
+                    </p>
+            </li>
+            <li class="flex-item">
+                <h1 style="margin-bottom: 0px;">VOLK</h1>
+                <h4>(SIMD Optimization)</h4>
+                <div style="text-align: left;">
+                    <p style="line-height: 1.4;">
+                    Lead:<br>
+                    Johannes Demel<br><br>
+                    Members:<br>
+                    Michael Dickens<br>
```

Comment:
I always consider @michaelld and myself to co-lead VOLK. Thus, I suggest to put both names in the "leads" line.

Reply:
separately we should all decide whether we want co-leads allowed, pros/cons of either way, but we can fix it in a different PR

Reply:
When we opted into leading Volk, neither one of us wanted to be the sole lead; we were OK with co-leading. That's still the case for me.

As I did not even know that I was listed as "member" and not "co-lead" _before_ this PR, and all that's being updated is the email address, I'm OK with moving the "co-lead" change to another PR. That said, now that I _do_ know, so long as @jdemel and I consider each other to be co-leads and neither wants to be the sole lead (even with backup from the other), then I will insist on being listed as such.

The above stated: I don't see why we can't also updated this listing via another commit to this PR ... here and in "layouts/org/single.html" that's being added (above). Just get it done.